pkg/netceptor/netceptor.go

```diff
@@ -902,13 +902,13 @@ func (s *Netceptor) flood(message []byte, excludeConn string) {
 	defer s.connLock.RUnlock()
 	for conn, ci := range s.connections {
 		if conn != excludeConn {
-			go func(ci *connInfo) {
+			go func(conn string, ci *connInfo) {
 				select {
 				case ci.WriteChan <- message:
 				case <-ci.Context.Done():
-					s.Logger.Debug("connInfo cancelled during flood write")
+					s.Logger.Debug("connInfo for connection %s cancelled during flood write", conn)
```

Comment:
Would `s.NodeID()` be helpful in this log also @fosterseth ?

Reply:
probably not necessary in this case